dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -422,6 +422,7 @@ class CompileTest {
       case DeviceOperatingSystem.android:
         options.insert(0, 'apk');
         options.add('--target-platform=android-arm');
+        options.add('--tree-shake-icons');
```

Comment:
How about using '--tree-shake-icons' or '--tree-shake-fonts' and Icon/FontTreeShaker everywhere instead of FontSubset?

Reply:
Done